### ISSUE TITLE
Errors in ver 3

### ISSUE THREAD

**Issue by andrew-azarov** [flan/staticdhcpd#95]
After upgrade to branch 3 got errors in:

```
Traceback (most recent call last):
  File "/usr/local/bin/staticDHCPd", line 297, in <module>
    _initialiseDHCP()
  File "/usr/local/bin/staticDHCPd", line 226, in _initialiseDHCP
    import staticdhcpdlib.dhcp
  File "/usr/local/lib/python3.10/dist-packages/staticdhcpdlib/dhcp.py", line 35, in <module>
    import libpydhcpserver.dhcp
  File "/usr/local/lib/python3.10/dist-packages/libpydhcpserver/dhcp.py", line 320
    def _setupListeningSockets(self, server_port, proxy_port server_address=None, link_local_only=False):
                                                             ^^^^^^^^^^^^^^
SyntaxError: invalid syntax
```
Another error

```
2023-09-07 08:53:11,310 : CRITICAL : System shutdown triggered by unhandled exception:
Traceback (most recent call last):
  File "/usr/local/bin/staticDHCPd", line 297, in <module>
    _initialiseDHCP()
  File "/usr/local/bin/staticDHCPd", line 227, in _initialiseDHCP
    dhcp = staticdhcpdlib.dhcp.DHCPService(database)
  File "/usr/local/lib/python3.10/dist-packages/staticdhcpdlib/dhcp.py", line 850, in __init__
    self._dhcp_server = _DHCPServer(
  File "/usr/local/lib/python3.10/dist-packages/staticdhcpdlib/dhcp.py", line 403, in __init__
    libpydhcpserver.dhcp.DHCPServer.__init__(
  File "/usr/local/lib/python3.10/dist-packages/libpydhcpserver/dhcp.py", line 117, in __init__
    self._network_link = _NetworkLink(str(server_address), server_port, client_port, proxy_port, response_interface, response_interface_qtags=response_interface_qtags, link_local_only=local_service_only)
NameError: name 'local_service_only' is not defined
```

**Comment by andrew-azarov** [comment]
https://github.com/flan/staticdhcpd/pull/96

**Comment by flan** [comment]
Marking this as closed after merging #96 (https://github.com/flan/staticdhcpd/commit/2632b997362ec71c0662b5a39e46574b2a15b319), thanks to a patch from Andrew.
